WooCommerce/Classes/ViewRelated/AppCoordinator.swift

```diff
@@ -250,7 +253,7 @@ private extension AppCoordinator {
                 "action": "contact_support",
                 "type": response.notification.request.identifier
             ])
-        default:
+        case UNNotificationDefaultActionIdentifier:
```

Comment:
It feels like this `func handleLocalNotificationResponse` method is growing large to handle many types of local notifications. If you feel the same way, perhaps we should try to think of a way to break this down into multiple private methods. (In a future PR 😅 )

Reply:
Thanks for the suggestion, I'll keep this in mind the next time touching this switch statement 🙂